### PR TITLE
Apply black style to test_Ge* files, version 19.10b0.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to test_Ge* files, files listed below; should be fine to merge.
test_GenBank.py
test_GenomeDiagram.py

### CODE REVIEW

File: Tests/test_GenomeDiagram.py [hunk 1]
```diff
@@ -213,14 +218,18 @@ def test_color_conversions(self):
...
+        ), "Did not correctly translate colour from floating point RGB tuple"
+        assert translator.int255_color((1, 75, 240)) == translator.translate(
+            (1, 75, 240)
+        ), "Did not correctly translate colour from integer RGB tuple"
+        assert translator.artemis_color(7) == translator.translate(
+            7
+        ), "Did not correctly translate colour from Artemis colour scheme"
+        assert translator.scheme_color(2) == translator.translate(
+            2
+        ), "Did not correctly translate colour from user-defined colour scheme"
```
Comment (peterjc) [inline Tests/test_GenomeDiagram.py]: These assert statements look ugly - they should be using ``self.assertEqual(...)`` instead which will look nicer.

**Comment by peterjc** [comment]
There is no connection between these two files other than the first two letters - I don't think it is a very useful grouping in a single commit.

Interestingly the GenBank tests were already run through black v19.3b0, I would prefer to have that as a single commit (with a comment about this).

Are you comfortable with ``git rebase`` or otherwise changing the branch history? If not, maybe split this into two pull requests?

**Comment by svalqui** [comment]
I re-do this one, 2 PR and I'll close this.

**Comment by peterjc** [comment]
With #2766 applied, this looks better.

**Comment by peterjc** [comment]
If it is easier to make another pull request, that's fine too.

P.S. Does the ``self.assertEqual(...)`` change make sense?

File: Tests/test_GenomeDiagram.py (hunk 1 above)
Comment (svalqui) [reply Tests/test_GenomeDiagram.py]: Done on #2767 , thank you, I'll close this one.

**Comment by svalqui** [comment]
closing this, created #2766 and #2767 instead.

**Pull request closed without merging**